Tests/iaas/flavor-naming/flavor-name-check.py

```diff
@@ -27,11 +27,14 @@
 import sys
 import os
 import re
+# import getopt
```

Comment:
unused, commented out import

Reply:
Removed.
Will readd in a subsequent patch, as the hand-written option parsing needs to go.